src/libraries/Microsoft.PowerFx.Core/Public/Values/CompileTimeTypeWrapperRecordValue.cs

```diff
@@ -27,17 +30,24 @@ private CompileTimeTypeWrapperRecordValue(RecordType type, RecordValue inner)
         }
 
         protected override bool TryGetField(FormulaType fieldType, string fieldName, out FormulaValue result)
+        {
+            (var res, result) = TryGetFieldAsync(fieldType, fieldName, CancellationToken.None).Result;
```

Comment:
>(var res, result) [](http://example.com/codeflow?start=12&length=17)

This syntax is not supported in VS2019 (our current min version); rewriting it as follows works:

            var (tryGetResult, value) = TryGetFieldAsync(fieldType, fieldName, CancellationToken.None).Result;
            result = value;
            return tryGetResult;